sklearn/titanic/README.md

```diff
@@ -0,0 +1,99 @@
+## Introduction
+
+This sample code illustrates how to start a GCE instance and train a model using scikit-learn on the instance. We will be using the [Titanic dataset](https://www.kaggle.com/c/titanic) from [Kaggle](https://www.kaggle.com/). The goal is to train a classifier to predict whether a given passenger on Titanic survived or not.
+
+## Prerequisites
+
+You need to download [train.csv](https://www.kaggle.com/c/titanic/download/train.csv) from [Kaggle](https://www.kaggle.com/).
+
+The sample assumes that you already have a [GCP](https://cloud.google.com/) account and you have [downloaded](https://cloud.google.com/sdk/), installed, and [configured](https://cloud.google.com/sdk/gcloud/reference/config/) it on your computer.
+
+Finally, you need to have a project created on GCP. For further instructions, please check [here](https://cloud.google.com/sdk/gcloud/reference/projects/create).
+
+
+## Steps:
+We will create a Google Compute Engine instance, configure it, and use it to train a model. While some of the steps can be done using the web portal [here](https://pantheon.corp.google.com), we will try to accomplish this using the SDK and mainly the [gcloud](https://cloud.google.com/sdk/gcloud/) command.
+
+##### 1. Create a Google Compute Engine Instance
+First, we should create a GCE instance that we can use to train our model on. In this case, we can use many of the default arguments and simply create the instance:
+
+```
+gcloud compute instances create MYINSTANCE
+```
+
+where MYINSTANCE is our given name to this instance. By default, this instance will be of type *n1-standard-1* with 1 CPU and 3.75GB of RAM. To see a list of available machine types, you can use:
```

Comment:
@happyhuman : Could you also link to the page listing instance types here?

URL: https://cloud.google.com/compute/docs/machine-types